data_prep/roberta_graph_dataset.py

```diff
@@ -45,7 +45,7 @@ def _generate_features(self):
                 docs = self._raw_texts[i * batch_size:max_docs]
                 encodings = self._tokenizer(docs, truncation=True, padding=True)
                 items = {k: torch.tensor(v, dtype=torch.long, device=self._device) for k, v in encodings.items()}
```

Comment:
I don't really know why you added the creation of this dictionary. I don't think we need it as my previous code worked fine but I guess you put this here for a reason so I left it in. I had to change the output of the doc_embedder though because we only need the final hidden states

Reply:
Because we are using our `RobertaEncoder` model, with needs both, the `input_ids` and the `attention_mask`. Both is contained in the encodings returned by the tokenizer.

Reply:
I see you are not using our `RobertaEncoder` anymore, then this dict creating can also be removed again.